salt/preflight/defaults.yaml

```diff
@@ -0,0 +1,32 @@
+destructive: False      # define if /var/lib/etcd if should be removed
```

Comment:
Generic comment: you'll want to rebase on `development/2.0` and move this state into e.g. `salt/metalk8s/kubeadm/init/preflight/` or so.